.delivery/build-cookbook/recipes/deploy.rb

```diff
@@ -75,27 +75,6 @@
   end
 end
 
-# Resources for managing old instances
-# These will eventually be destroyed
-# Keeping them to prevent downtime
```

Comment:
don't be alarmed. comment referenced AUR environments. Delivered remain unscathed.